OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1888 @@
+    e.AddComponent<ConvexMeshCollider3DComponent>();
+    e.AddComponent<TriangleMeshCollider3DComponent>();
+    e.AddComponent<DirectionalLightComponent>();
+    e.AddComponent<PointLightComponent>();
+    e.AddComponent<SpotLightComponent>();
+    e.AddComponent<ScriptComponent>();
+    e.AddComponent<LuaScriptComponent>();
+    e.AddComponent<ModelComponent>();
+
+    auto checkHas = [&](const char* name)
+    {
+        auto r = lua.script(std::string("return entity_utils.has_component(eid, '") + name + "')");
```

Clang (14.0.6)

[warning] variable name 'r' is too short, expected at least 3 characters

(readability-identifier-length)